cli/cli.js

```diff
@@ -107,25 +108,41 @@ async function pouchMigrate() {
 
     console.log(await remote.info());
 
-    await remote.destroy();
+    const documents = await remote.allDocs({ include_docs: true });
+    const crops = documents.rows.map(document => document.doc);
+    const binomialNameToCrop = mapCropsByBinomialName(crops);
```

Comment:
I would suggest in this case to not allow editing of binomial name. As in case of editing binomial name, after rerun of migration this part will create new record with old binomial name so there will be kind of duplicate entry.

Reply:
I opened #85 for redefining and specifying the migration process.

The migration/initialization process would be somewhat easier to implement if the initial dataset contained IDs, but I prefer to keep `practicalplants.json` database agnostic so in my opinion there shouldn't be IDs in this file (IDs could be removed during #70).